frontend/src/components/TableComponents/TableComponents.tsx

```diff
@@ -70,8 +70,8 @@ export const Row = ({
       if (typeof cell === 'string') {
         return <Cell key={idx} data={cell} />;
       }
-      const { data: cData, tag, smallTag } = cell;
-      return <Cell key={idx} data={cData} tag={tag} smallTag={smallTag} />;
+      const { data: cData, tag } = cell;
+      return <Cell key={idx} data={cData} tag={tag} smallTag={true} />;
```

Comment:
Is it still possible to set tags as not reduced? Or this functionality is just abandoned?

Reply:
abandoned for now, see my comment below